app/scripts/views/settings/avatar_crop.js

```diff
@@ -37,6 +38,12 @@ function (p, Cocktail, FormView, SettingsMixin, AvatarMixin, Template,
       }
     },
 
+    events: {
+      'click #back': function () {
```

Comment:
This seems [like a duplicated pattern](https://github.com/mozilla/fxa-content-server/pull/2748/files#diff-e8bcfe428107ce33dbc222423506acd2R39), should it somehow be part of the ModalSettingsPanelMixin where the URL to navigate to is somehow figured out programatically? What happens if you just go back in history?